FindRange.java

```diff
@@ -9,8 +9,24 @@
 import acm.program.*;
```

Comment:
Great solution. However you missed one edge case:

If the user enters the sentinel on the very first input line, then no values have been
entered, and your program should display a message to that effect.
```
So if for example the user enters 0 the program should report:
No values has been entered.
Your program instead reports:
Smallest: 0
Largest: 0